CONTRIBUTING.md

```diff
@@ -1,14 +1,14 @@
 Contributing to PrestaShop
 --------------------------
 
-PrestaShop is an open-source e-commerce solution. Everyone is welcome and even encouraged to contribute with their own improvements.
+PrestaShop is an Open Source ecommerce solution. Everyone is welcome and even encouraged to contribute with their own improvements.
```

Comment:
It's "open source" or "open-source" but not "Open Source". It's not a name like Open Source Foundation or Open Source Initiative

Reply:
Yeah @PierreRambaud :blush: Your right